sklearn/cross_decomposition/_pls.py

```diff
@@ -467,8 +470,8 @@ def predict(self, X, copy=True):
         # Normalize
         X -= self._x_mean
         X /= self._x_std
-        Ypred = X @ self.coef_.T
-        return Ypred + self.intercept_
+        Ypred = X @ self.coef_.T + self.intercept_
+        return Ypred.ravel() if self._predict_1d else Ypred
```

Comment:
nit: my personal taste/preference is to split this up into a `if self._predict_1d: ... else: ...` (so spread over four lines). I find it easier to read/spot that there is a `if` here compared to using the one liner. But others might have other tastes :-/

Reply:
In my opinion I think it is fine both ways.